.docksal/commands/refresh

```diff
@@ -1,30 +1,33 @@
 #!/usr/bin/env bash
 
-#: exec_target = cli
-
-## Initialize stack and site (full reset)
+## Refresh DB
 ##
 ## Usage: fin refresh [environment id]
+## Defaults to the primary environment
+
+# Abort if anything fails
+set -e
+
+ENVIRONMENT=${1:-"$hostingenv"}
+DOCROOT_PATH="${PROJECT_ROOT}/${DOCROOT}"
 
-ENVIRONMENT=${1:-prod}
+echo "Pulling Database..."
+fin pull db -y --hosting-env=${ENVIRONMENT}
 
-cd /var/www/${DOCROOT}
+echo "Update code through composer"
```

Comment:
This is a pretty drastic change to the refresh command.  Usually we only have it get the database.

Reply:
So my take on that is if I switch branches I want to make sure I can run the DB for that specific branch so I need to make sure I have any module or patch changes applied automatically.   it doesn't upgrade anything it just installs what's in composer.json

Reply:
This also allows a pretty close to release to prod test locally.   Download the DB, update the code and then run the DB updates, etc

Whenever the support team gets a refresh script without composer install this gets added because the refresh process should help us test what it's going to be like releasing the branch against production